src/tools/rbd_mirror/InstanceReplayer.cc

```diff
@@ -359,12 +380,15 @@ void InstanceReplayer<I>::start_image_replayers(int r) {
     start_image_replayer(current_it->second);
   }
 
-  m_service_daemon->add_or_update_attribute(
-    m_local_pool_id, SERVICE_DAEMON_ASSIGNED_COUNT_KEY, image_count);
-  m_service_daemon->add_or_update_attribute(
-    m_local_pool_id, SERVICE_DAEMON_WARNING_COUNT_KEY, warning_count);
-  m_service_daemon->add_or_update_attribute(
-    m_local_pool_id, SERVICE_DAEMON_ERROR_COUNT_KEY, error_count);
+  // TODO: add namespace support to service daemon
```

Comment:
Might be nice to have it automatically aggregate the stats from namespaces (and default namespace) to the high-level stats.

Reply:
Probably yes. I think it can be done later as a separated PR.